### PR TITLE
Adding my favorite song

### PR DESCRIPTION
**Pull request by fredrhaetw**

### CODE REVIEW

**Comment by Luxygit** [comment]
I don't like that song

File: other-file.txt [hunk 1]
```diff
@@ -3,4 +3,4 @@ Congrats, you found the OTHER file!
 There's another question in this one, but I promise it's not too tricky!
 
 Question: What is your favourite song?
-Your answer:
\ No newline at end of file
+Your answer: Quiet Little Voices (We were promised jetpacks)
```
Comment (Elyamira) [inline other-file.txt:6]: Nice song!

Comment (fredrhaetw) [reply other-file.txt:6]: Great! You have a nice taste.

`A better way to write comments with code`

Comment (Luxygit) [reply other-file.txt:6]: I still don't like it.

**Pull request merged**